lib/packages/core/README.md

```diff
@@ -16,6 +20,20 @@ or using [yarn](https://yarnpkg.com/):
 yarn add @burstjs/core
 ```
 
+### Using in classic `<script>`
+
+
+
+```js
+// using core
+const api = b$.composeApi({
+  nodeHost: "http://at-testnet.burst-alliance.org:6876",
+  apiRootUrl: "/burst"
+});
+
+api.network.getBlockchainStatus().then(console.log);
+```
+
 ## API Reference
```

Comment:
Also, this file is generated. Did you update the template?

Reply:
Oops. Nope. Docs will be updated in upcoming PR.